mappings/net/minecraft/client/gui/screen/container/ContainerScreen.mapping

```diff
@@ -5,6 +5,7 @@ CLASS id net/minecraft/client/gui/screen/container/ContainerScreen
 	FIELD l itemRenderer Lbb;
 	METHOD <init> (Ldw;)V
 		ARG 1 container
+	METHOD a drawBackground (F)V
```

Comment:
Is it better to be consistent and call `draw` to `render` like `renderContainerBackground` as `renderBackground` is already is thing?

Reply:
```suggestion
	METHOD a renderContainerBackground (F)V
```